RUFAS/data_validator.py

```
@@ -1755,6 +1755,46 @@ def extract_value_by_key_list(
+        ValueError
+            If any of the following rules are violated:
+
+            - ``entry_weight`` must be > 0.
+            - ``slaughter_weight`` must be > ``entry_weight``.
+            - ``max_days_on_feed`` must be > 0.
+            - ``mud_condition`` must be one of ``'none'``, ``'mild'``, or ``'severe'``.
+
+        """
+        entry_weight = float(feedlot_config.get("entry_weight", 0))
+        if entry_weight <= 0:
+            raise ValueError(f"feedlot entry_weight must be > 0, got {entry_weight}")
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)